src/cljam/fasta/reader.clj

```diff
@@ -131,6 +121,9 @@
          (.clear ~buffer)
          ba#)))
 
+(def ^:private gt-byte (byte \>))
+(def ^:private newline-byte (byte \newline))
```

Comment:
Thanks to your review. I fixed it by https://github.com/chrovis/cljam/pull/51/commits/3c4fcdd4574e3cd70e130d30e50cffe3369369d6 .